src/macaron/slsa_analyzer/provenance/intoto/v01/__init__.py

```diff
@@ -39,10 +39,6 @@ def validate_intoto_statement(payload: dict[str, JsonType]) -> TypeGuard[InTotoV
 
     Specification: https://github.com/in-toto/attestation/tree/main/spec/v0.1.0#statement.
 
-    TODO: Consider using the in-toto-attestation package (https://github.com/in-toto/attestation/tree/main/python),
```

Comment:
I have a question about this: I vaguely remember that you mentioned looking at this package and seeing some issues with it. I wonder what was the issue you saw and whether we want to reconsider using this package again in the future.

Reply:
It does not provide any methods relating to verifying the validity of a given attestation file in the straightforward way that we require.